packages/manager/modules/pci/src/projects/project/storages/databases/database.service.js

```diff
@@ -0,0 +1,303 @@
+import filter from 'lodash/filter';
+import find from 'lodash/find';
+import get from 'lodash/get';
+import map from 'lodash/map';
+import set from 'lodash/set';
+import 'moment';
+
+import Backup from '../../../../components/project/storages/databases/backup.class';
+import Database from '../../../../components/project/storages/databases/database.class';
+import Engine from '../../../../components/project/storages/databases/engine.class';
+
+export default class DatabaseService {
+  /* @ngInject */
+  constructor($http, $q, $translate, CucPriceHelper) {
+    this.$http = $http;
+    this.$q = $q;
+    this.$translate = $translate;
+    this.CucPriceHelper = CucPriceHelper;
+  }
+
+  addNode(projectId, engine, databaseId, flavor, region) {
+    return this.$http
+      .post(
+        `/cloud/project/${projectId}/database/${engine}/${databaseId}/node `,
```

Comment:
can you extract dynamic API url "/cloud/project/${projectId}/database/${engine}/${databaseId}" into a separate method and use it wherever required? looks like you are using it multiple places.

getDatabaseBaseUrl(projectId, engine, databaseId) {
return `/cloud/project/${projectId}/database/${engine}/${databaseId}`
}

Reply:
Done